core/src/main/resources/libs/howlite/components/footer/dialog/.content.json

```diff
@@ -107,6 +107,11 @@
             "label": "Instagram",
             "value": "icon-instagram"
           }
+        },
+        "nofollow": {
+          "sling:resourceType": "wcm/dialogs/components/toggle",
```

Comment:
I think it would be better to use a checkbox here. From the UX perspective toggles are used usually to manipulate dialog, so together with some display conditions.